Lib/asyncio/events.py

```diff
@@ -782,7 +797,10 @@ def get_event_loop():
     the result of `get_event_loop_policy().get_event_loop()` call.
     """
     # NOTE: this function is implemented in C (see _asynciomodule.c)
-    return _py__get_event_loop()
+    current_loop = _get_running_loop()
+    if current_loop is not None:
+        return current_loop
+    return get_event_loop_policy().get_event_loop()
 
 
 def _get_event_loop(stacklevel=3):
```

Comment:
Should we add a comment here that this is no longer used, will disappear in 3.12, but is retained in 3.10-3.11 in case some user code was calling it? (Which I think is why we are keeping it.)

Reply:
Done.

Reply:
Yup, will do.